attributes/default.rb

```diff
@@ -4,6 +4,15 @@
 # All rights reserved.
 #
 
+default['newrelic_infra']['provider'] = 'package_manager'
+default['newrelic_infra']['tarball']['version'] = nil
+
+case node['kernel']['machine']
+when 'x86_64'
+  default['newrelic_infra']['tarball']['architecture'] = 'amd64'
+when 'i386'
+  default['newrelic_infra']['tarball']['architecture'] = '386'
```

Comment:
is this needed? I only saw 64-bit tarballs

Reply:
I think the point of this is to be able to use chef to install tarballs of different architectures. We will provide tarballs for different architectures, and the role of this config management tools is to abstract this as much as possible. I had to make this `case` because what chef provides is not exactly the same as the names we use. I couldn't find what is returned for the other archs so we can leave it like this and keep adding as we go.

I realized I missed the default case, I'm doing a commit with the changes.